packages/block-library/src/post-comments-count/index.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-comments-count` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-comments-count` block on the server.
+ *
+ * @return string Returns the filtered post comments count for the current post.
+ */
+function render_block_core_post_comments_count() {
+	$post = gutenberg_get_post_from_context();
+	if ( ! $post ) {
+		return '';
+	}
+	return get_comments_number( $post );
```

Comment:
I believe this should have the same span (and className) used in edit right?

Reply:
Do we have to register `className` as an attribute in the server for this?

Reply:
That's possible. I honestly don't remember exactly.

Reply:
Looking at the "latest posts" block, it seems the block classname `wp-block-latest-posts` is added manually on the server and the attribute need to be registered for the custom className support.

I blieve the className for this block is `wp-block-post-comments-count`

Reply:
Done :+1: